.pre-commit-config.yaml

```diff
@@ -53,6 +53,9 @@ repos:
     hooks:
       - id: ruff-format
       - id: ruff
+        args:
+          - --extend-ignore=ISC001
```

Comment:
```
ruff-format..............................................................Failed
- hook id: ruff-format
- files were modified by this hook

warning: The following rule may cause conflicts when used with the formatter: `ISC001`. To avoid unexpected behavior, we recommend disabling this rule, either by removing it from the `select` or `extend-select` configuration, or adding it to the `ignore` configuration.
```

Reply:
Ignores are managed [here](https://github.com/luminartech/dev-tools/blob/8ac690a3515a7922b96ad98d9d5b5a4203bd4c76/pyproject.toml#L88-L99)